docs/Home.md

```diff
@@ -1,3 +1,50 @@
-Welcome to the android-templates wiki! 
+Welcome to the android-templates wiki!
 
-More coming soon...
\ No newline at end of file
+This repository generates a new project based on our preferences, by running a simple script. 
+
+Ex: `./newproject.sh -t crt -p co.myproject.example -n "MyProjectExampleCoroutine"`
+
+This script must include all essentials by default, while optional features can be appended with flags. To prevent this repository from becoming a dumping playground, features can be rejected too.
+
+Planning to contribute? Let's have a look at the criteria
+
+How do we decide if a feature is **essential**? 👍
+
+- Our team requires it to maintain code quality
+- Our team requires it to maintain a good developer experience
+- Our team requires it to increase the productivity of the team
+- It is always used or configured in the projects we've worked on
+- It is recommended to use or configure based on Google's standards
+
+How do we decide if a feature is **optional**? 🚩
+
+- It is regularly used or configured in projects we've worked on
+- It is regularly used or configured as an alternative to the default choice
+
+How do we decide if a feature is **rejected**? 👎
+
+- It is barely used or configured in projects we've worked on
+- It is barely used or configured as an alternative to the default choice
+
+Let's have a look at some examples 🔎
+
+**Essential**:
+- [Jacoco](https://github.com/jacoco/jacoco): Code coverage is important to maintain code quality.
+- [Hilt](https://developer.android.com/training/dependency-injection/hilt-android): Dependency injection provides a good developer experience.
+- [Firebase App Distribution](https://firebase.google.com/docs/app-distribution): Continuous delivery increases the productivity of the team.
+- [Glide](https://github.com/bumptech/glide): Image loading is always used in all the projects we've worked on.
+- [Compose](https://developer.android.com/jetpack/compose): A modern toolkit for building native UI which is recommended by Google
+
+**Optional**:
+- [Lottie](https://github.com/airbnb/lottie-android): Loading animations with a JSON file is regularly used, but not every project needs it.
+- [Bitrise](https://www.bitrise.io/): A CI/CD provider which is regularly used, but [GitHub Actions](https://github.com/features/actions) is our default choice.
```

Comment:
I wonder about our default choice, do we include the default choice to the `essential` too?

Reply:
No, we don't need to add the default choice to `essential`, because all the essentials are our default already 🙏 

I've added the default choice here as an example, to prove that Bitrise is an `optional`

Reply:
@Tuubz Imo, it would be more understandable if we separate them into: 

- `Must Have` - which cannot be missing and or replaceable.

- `Nice to have` - which can be replaceable, but so far, it is our best choice.

- `Optional` - which can be replaced by other ones anytime.

Reply:
@doannimble Please have a look at [this thread](https://nimble-co.slack.com/archives/G41UCTGHL/p1642578956011700?thread_ts=1642573191.009000&cid=G41UCTGHL) where we discuss the naming, hopefully, it can clear up the naming choice  🙏